Swift/Tests/URLEndpointListenerTest.swift

```diff
@@ -791,7 +791,7 @@ class URLEndpointListenerTest_Main: URLEndpointListenerTest {
         try stopListener()
     }
     
-    func testEmptyNetworkInterface() throws {
+    func _testEmptyNetworkInterface() throws {
```

Comment:
This seems to be unrelated to this PR. Is this by intention?

Reply:
accidental